src/index.ts

```diff
@@ -72,6 +74,8 @@ if (window) {
     CamerasPosition,
     RealtimeComponentState,
     RealtimeComponentEvent,
+    ComponentLifeCycleEvent,
+    WhoIsOnlineEvent,
```

Comment:
```suggestion
    ComponentLifeCycleEvent,
    WhoIsOnlineEvent,
    CommentEvent,
```

Reply:
`CommentEvent` is a few lines above in the original code, in line 55